### PR TITLE
Enhancement/16136-popup-button-styles

### PR DESCRIPTION
**Pull request by pawellysy**
Added new styles for buttons in Stock Tools popup, see #16136.

### CODE REVIEW

File: css/annotations/popup.scss [hunk 1]
```diff
@@ -16,6 +16,7 @@ $white: #fff;
 
 .highcharts-popup.highcharts-annotation-toolbar button {
   margin-top:0px;
+  color: blue;
```
Comment (raf18seb) [inline css/annotations/popup.scss:19]: `blue` or `$blue`?

File: css/annotations/popup.scss [hunk 2]
```diff
@@ -180,21 +181,28 @@ $white: #fff;
...
+  border-width: 0;
   cursor: pointer;
 }
 
 .highcharts-popup button {
   float: right;
   border: none;
-  background: $button-background-color;
-  color: #666;
+  background: $blue;
```
Comment (raf18seb) [inline css/annotations/popup.scss:194]: Shouldn't is be `background-color` instead of `background`?

File: css/annotations/popup.scss [hunk 3]
```diff
@@ -180,21 +181,28 @@ $white: #fff;
 }
 
 .highcharts-popup-close:hover,
-.highcharts-popup button:hover,
 .highcharts-popup button.highcharts-annotation-edit-button:hover,
 .highcharts-popup button.highcharts-annotation-remove-button:hover {
   background-color: $button-hover-color;
+  border-width: 0;
```
Comment (raf18seb) [inline css/annotations/popup.scss:187]: Shouldn't be `border: none;`?

**Review by raf18seb — CHANGES_REQUESTED** [review]

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
PR:
Screenshot 2021-08-17 at 10 40 10

Mockup: 
128012324-db45ab20-34d7-455a-ace6-5655721a6279

My only concern is with the size of the add/save/remove button.
I think it looks bigger in the mockup, but I'm not sure if this was part of that task.
IMO the padding should be increased.

File: css/annotations/popup.scss [hunk 4]
```diff
@@ -1,7 +1,8 @@
 // Colors for buttons.
 $button-background-color: #f7f7f7;
 $button-hover-color: #e6ebf5;
-
+$blue: #424BCD;
+$white: #fff;
```
Comment (pawelfus) [inline css/annotations/popup.scss:5]: While `$white` is fine, `$blue` is not necessarily the best description.

File: css/annotations/popup.scss [hunk 5]
```diff
@@ -238,4 +251,32 @@ $button-hover-color: #e6ebf5;
...
+  display: block;
+  float: none;
+}
+
+.highcharts-popup button.clear-filter-button {
+  margin: 0 auto;
+  display: block;
+  float: unset;
+  color: $blue;
+  background-color: #fff;
```
Comment (pawelfus) [inline css/annotations/popup.scss:277]: *white

File: css/annotations/popup.scss [hunk 6]
```diff
@@ -238,4 +251,32 @@ $button-hover-color: #e6ebf5;
...
+  border: 1px solid $blue;
+}
+
+.highcharts-popup button.clear-filter-button:hover {
+  color: #fff;
```
Comment (pawelfus) [inline css/annotations/popup.scss:282]: *white

**Review by pawelfus — CHANGES_REQUESTED** [review]
It's a bit tricky to review only changes related to the ticket. Shall we review this once again, when the main PR will get merged, or PR this into main branch (instead of the master)?

**Review by raf18seb — APPROVED** [review]

File: css/annotations/popup.scss (hunk 4 above)
Comment (pawellysy) [reply css/annotations/popup.scss:5]: is `$dark-blue` ok?

File: css/annotations/popup.scss [hunk 7]
```diff
@@ -15,6 +16,7 @@ $button-hover-color: #e6ebf5;
 
 .highcharts-popup.highcharts-annotation-toolbar button {
   margin-top:0px;
+  color: $blue;
```
Comment (pawelfus) [inline css/annotations/popup.scss:19]: `$blue` or `$dark-blue`? There's more of it (check the whole file)

**Review by pawelfus — CHANGES_REQUESTED** [review]

**Review by karolkolodziej — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Looks good to me, mending approval by @pawelfus

**Comment by TorsteinHonsi** [comment]
Bump @pawelfus

**Comment by pawelfus** [comment]
Sorry, I'm waiting for https://github.com/highcharts/highcharts/pull/16089 to be finished and merged. There's a lot of noise from the other branch, so it's hard to tell what is related to the ticket and what is not

**Pull request closed without merging**